students/KyleBarry/session04/trigrammify.py

```diff
@@ -0,0 +1,60 @@
+import random
+import requests
+from bs4 import BeautifulSoup
+
+
+def get_text(url): 
+    """ 
+    Get text data from url of gutenberg project so as to not have it within code
+    """
+    page = requests.get(url)
+    soup = BeautifulSoup(page.content, "lxml")
```

Comment:
getting fancy!

butnote that project gutenbuerg also has plain tet download:

http://www.gutenberg.org/files/2701/2701-0.txt

So you could have done that a bit simpler :-)

(check out the "requests" module for quick and easy downloading -- `pip install requests`)

Also -- Charles had problems with BeautifulSoup -- so couldn't run your code -- not your problem , but it is a reminder why it's good to use as simple as possible tools :-)